views/pages/logout.php

```diff
@@ -0,0 +1,7 @@
+<?php
```

Comment:
Hey Isaac, since this page doesn't render any HTML (the user would never seen any output from this file), the code more naturally belongs in the `controllers` directory. In fact, we already have this code in the following file: `controllers/authentication/logout-controller.php`

Let me know if you think there's something I'm missing.